### PR TITLE
symmetric stepping instructions

### PR DESCRIPTION
**Pull request by IvicaL**

### CODE REVIEW

File: spyboat/phase_fields/gradient_descent.py [hunk 1]
```diff
@@ -21,46 +21,92 @@
...
+    22.5, 67.5, 112.5, 157.5, 180])
+    
+    crawl_angle = np.arctan2(vy, vx)* 180 / np.pi
+
+    # directions = np.array(['left', 'left_down', 'down', 'right_down',\
+    #  'right','right_up', 'up', 'left_up', 'left'])
+    directions = np.array([(-1,0),(-1, -1),(0, -1), (+1, -1),\
+    (+1, 0), (+1, +1), (0, +1), (-1, +1), (-1, 0)])
+
+    instruction = directions[np.where(crawl_angle<angle_boxes)][0]
```
Comment (tensionhead) [inline spyboat/phase_fields/gradient_descent.py]: Here we could be more explicit by 1st getting the index of the 'directions', the last `[0]` is confusing

File: spyboat/phase_fields/gradient_descent.py [hunk 2]
```diff
@@ -21,46 +21,92 @@
...
         y_old = y_new
 
-    xs, ys = list(zip(*xys))
+        x_o = x_n
+        y_o = y_n
 
-    return xs, ys
+    xs, ys = list(zip(*xys))
+    xs2, ys2 = list(zip(*xys2))
+    return xs, ys, xs2, ys2
```
Comment (tensionhead) [inline spyboat/phase_fields/gradient_descent.py:93]: what exactly are you returning here?

Comment (tensionhead) [reply spyboat/phase_fields/gradient_descent.py:93]: Ok got it..

File: spyboat/phase_fields/gradient_descent.py (hunk 1 above)
Comment (IvicaL) [reply spyboat/phase_fields/gradient_descent.py]: done

**Pull request merged**